scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -181,14 +184,25 @@ abstract class ScanResultsStorage {
      * Read all [ScanResult]s for a package with [id] from the storage. Return a [ScanResultContainer] wrapped in a
      * [Result], which is a [Failure] if no [ScanResult] was found and a [Success] otherwise.
      */
-    fun read(id: Identifier): Result<ScanResultContainer> =
-        readFromStorage(id).also {
-            stats.numReads.incrementAndGet()
-            if (it is Success && it.result.results.isNotEmpty()) {
+    fun read(id: Identifier): Result<ScanResultContainer> {
+        val (result, duration) = measureTimedValue { readFromStorage(id) }
+
+        stats.numReads.incrementAndGet()
+
+        if (result is Success) {
+            if (result.result.results.isNotEmpty()) {
                 stats.numHits.incrementAndGet()
             }
+
+            log.perf {
+                "Read ${result.result.results.size} scan results for '${id.toCoordinates()}' from " +
+                        "${javaClass.simpleName} in ${duration.inMilliseconds}ms."
```

Comment:
Would it make sense to say : "from storage '${javaClass.simpleName}' in "
So that the log messages can be easier matched? (Same below)

Reply:
I didn't add "storage" because all implementations have that already in their name, e.g. `PostgresStorage` or `FileBasedStorage`.

Reply:
I asked that, so that it would with any class, also anymous ones or ones that are not in our codebase. Anyhow, let's keep it.